tests/unit/test_cluster.py

```diff
@@ -19,48 +19,15 @@
 
 from ._testing import _make_credentials
 
+from tests.unit.test_base_cluster import (
```

Comment:
what's this change for?

Reply:
An attempt at deduping the various cluster test classes. `_Instance` and `_Client` implementation are shared between testers, and constants are the same regardless of sync/async.

Reply:
From what I saw it seems pretty feasible to declare a global helper class for all testers and import `_Instance` and `_Client` from there, but this seemed like a good middle-ground towards that. Happy to make that change in a separate PR though, or to keep it as-is if there's a reason for the current implementation.

Reply:
nope sounds good to me! Thanks for the explanation